README.md

```diff
@@ -258,6 +258,21 @@ We're now a part of **[Leapcode](https://www.leapcode.io/)**. It helps you contr
 | <a href="https://twitter.com/Vinit_Shahdeo"><img src="https://raw.githubusercontent.com/vinitshahdeo/Water-Monitoring-System/master/assets/twitter.png" width="32px" height="32px"></a> <a href="https://www.facebook.com/vinit.shahdeo"><img src="https://raw.githubusercontent.com/vinitshahdeo/Water-Monitoring-System/master/assets/facebook.png" width="32px" height="32px"></a> <a href="https://www.linkedin.com/in/vinitshahdeo/"><img src="https://raw.githubusercontent.com/vinitshahdeo/Water-Monitoring-System/master/assets/linkedin.png" width="32px" height="32px"></a> |
 
 
+## Other contributors:
+
+### Credits goes to these people:✨
+
+<table>
```

Comment:
Please move this to the existing Contributors section.

Reply:
Please @vinitshahdeo  Tell me exaclty what to Do  I don't get it !!

Reply:
@rohank2502,
You can remove `## Other contributors:` and move this section - `### Credits goes to these people:✨` to line 249, that is, just above the *Admin* section.